Sources/Sentry/SentryHub.m

```diff
@@ -323,15 +327,10 @@ - (SentryScope *)getScope
     }
 }
 
```

Comment:
`bindClient` is part of the unified API.
Isn't this on the umbrella header?

I'm not saying it shouldn't be deleted, it's just something we need to consider with care.

Reply:
I just moved the `bindClient` a few lines up. See lines 309 - 311.